### DISCUSSION TITLE
Vercel image optimization with next/image

### DISCUSSION THREAD

**Discussion by Hanskrogh** [vercel/next.js#34550] [Help]
We have a nextjs application hosted on vercel that uses image optimization. Our bandwidth usage is insanely high, 3 people that was testing the site managed to use 3GB in 6 hours. It seems like the images servered with the image optimization counts in as bandwidth?after we turned it off, it seems like our site is not using any bandwidth at all. 

Is my assumption correct?

**Comment by 2stash** [comment]
@Hanskrogh Hi, can you describe how/where you are using image optimization and how you turned it off?
